test/java/com/rapleaf/jack/TestGenericQuery.java

```diff
@@ -153,111 +144,7 @@ public void testBasicQuery() throws Exception {
             .or(User.BIO.equalTo("Trader"), User.HANDLE.equalTo("A")))
         .fetch();
     assertEquals(3, results1.size());
-    assertEquals(Sets.newHashSet(userA.getId(), userB.getId(), userC.getId()), Sets.newHashSet(results1.getLongs(User.ID)));
-  }
-
-  @Test
```

Comment:
The `TestGenericQuery` file is very large. So I broke the code into two files, one just for the query logic, and one for `Record` and `Records`. These tests have been moved to `TestRecordAndRecords `.